cibyl/sources/source.py

```diff
@@ -62,15 +62,6 @@ def __init__(self, name: str, driver: str, url: str = None,
         self.url = url
         self.priority = priority
 
-    # pylint: disable=unused-argument
```

Comment:
Create a pull request for this change on its own. I think it would be good to have a discussion about it before removing the functions. From my side, I have implemented the 'connect' method on the Zuul source.

Reply:
Do you call the connect method of Source from the Zuul source? Because as it is now, it does nothing but log and as far as I can tell it's not used anywhere and is not really part of the sources interface like `get_jobs`.

Reply:
No, I do not call it but I have implemented it. I just wanted to raise the question if it was meant to be used at one point.

Reply:
I don't think it should be considered a part of the interface since not all sources have it. Jenkins does not use it and the Jenkins Job Builder source does not really even have anything to connect to, since it's parsing a git repo.